firebase-installations-interop/src/main/java/com/google/firebase/installations/internal/FidListener.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations.internal;
+
+import androidx.annotation.NonNull;
+
+/**
+ * <aside class="warning"><strong>Provides an inter-operational interface only</strong>; instead,
+ * use {@link FidListener} to call a listener when a Fid changes.</aside>
```

Comment:
It looks like this javadoc discourages use of this interface and then immediately suggests to use the same interface instead. Can you clarify the intent here?